Origin: uos
Last-Update: 2022-05-16

--- marisa-0.2.6.orig/include/marisa/base.h
+++ marisa-0.2.6/include/marisa/base.h
@@ -34,7 +34,7 @@ typedef uint64_t marisa_uint64;
     ( defined(__sparc__) && defined(__arch64__) ) || \
     ( defined(__riscv) && (__riscv_xlen == 64) ) || \
     defined(__mips64) || defined(__aarch64__) || defined(__s390x__) || \
-    defined(__alpha__)
+    defined(__alpha__) || defined(__sw_64__)
  #define MARISA_WORD_SIZE 64
 #else  // defined(_WIN64), etc.
  #define MARISA_WORD_SIZE 32
