From: liujianqiang <liujianqiang@uniontech.com>
Date: Mon, 11 Mar 2024 03:01:05 +0000
Subject: Add wordsize detection for sw64

---
 include/marisa/base.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/marisa/base.h b/include/marisa/base.h
index 2452a4a..f2f6d29 100644
--- a/include/marisa/base.h
+++ b/include/marisa/base.h
@@ -34,7 +34,7 @@ typedef uint64_t marisa_uint64;
     ( defined(__sparc__) && defined(__arch64__) ) || \
     ( defined(__riscv) && (__riscv_xlen == 64) ) || \
     defined(__mips64) || defined(__aarch64__) || defined(__s390x__) || \
-    defined(__alpha__) || defined(__loongarch64)
+    defined(__alpha__) || defined(__loongarch64) || defined(__sw_64__)
  #define MARISA_WORD_SIZE 64
 #else  // defined(_WIN64), etc.
  #define MARISA_WORD_SIZE 32
